javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -44,9 +44,12 @@ function AppNavbar() {
           <NavDropdown.Item as={Link} to="/statistics">
             Course Occupancy by Class Level
           </NavDropdown.Item>
-          <NavDropdown.Item as={Link} to="/statistics">
+          <NavDropdown.Item as={Link} to="/statistics/classSize">
             Average Class Size by Department
           </NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/statistics/courseCount">
```

Comment:
I don't think addition of this courseCount nav drop down item is valid. Consider removing